catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/operation/PostgreSqlTableOperations.java

```diff
@@ -74,11 +76,16 @@ protected String generateCreateTableSql(
       String comment,
       Map<String, String> properties,
       Transform[] partitioning,
+      Distribution distribution,
       Index[] indexes) {
     if (ArrayUtils.isNotEmpty(partitioning)) {
       throw new UnsupportedOperationException(
           "Currently we do not support Partitioning in PostgreSQL");
     }
+    Preconditions.checkArgument(
+        null == distribution || distribution == Distributions.NONE,
```

Comment:
Is it necessary to judge null here?

Reply:
I think it's necessray, user use table API may use `null` for distribution.

Reply:
Actually,  `null` has already been transformed into `Distributions.NONE` before reaching here through `CatalogOperationDispatcher`

Reply:
Sorry, I did not notice code in `CatalogOperationDispatcher`. I will remove null judgement here